### PR TITLE
FEAT: added get-started page and few importable css files

### PR DESCRIPTION
**Pull request by kotesh-arya**
Screenshot from 2022-02-06 22-42-21

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
i like the naming convention and variables you have created for repeated styles.code look clean to me

**Review by shashikantmaurya0007 — APPROVED** [review]
Looks good to merge

File: Get-Started/introduction.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (yashpatel1797) [inline Get-Started/introduction.html:7]: You can add author meta tag here.

File: css/sidebar.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    display: flex;
+    flex-direction: column;
+}
+
+.sidebar-item:hover {
+    color: navy;
+}
+.side-bar .link-active{
+    background-color: var(--primary-color);
+    color: white;
```
Comment (yashpatel1797) [inline css/sidebar.css]: be consistent on using variables.

**Review by yashpatel1797 — CHANGES_REQUESTED** [review]
The code looks good to me.

**Review by yashpatel1797 — APPROVED** [review]

File: Get-Started/introduction.html (hunk 1 above)
Comment (kotesh-arya) [reply Get-Started/introduction.html:7]: Sure, I will do that 👍🏻

File: css/sidebar.css (hunk 2 above)
Comment (kotesh-arya) [reply css/sidebar.css]: Sure 👍🏻

**Review by ayush65 — APPROVED** [review]

**Pull request merged**